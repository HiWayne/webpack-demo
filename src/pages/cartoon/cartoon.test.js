import renderer from "react-test-renderer";
import Cartoon from "./index";

test("Cartoon", () => {
  const component = renderer.create(<Cartoon />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  //   // manually trigger the callback
  //   tree.props.onMouseEnter();
  //   // re-rendering
  //   tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();

  //   // manually trigger the callback
  //   tree.props.onMouseLeave();
  //   // re-rendering
  //   tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();
});
