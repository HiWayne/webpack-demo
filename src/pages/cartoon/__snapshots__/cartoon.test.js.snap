// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cartoon 1`] = `
<section
  className="container"
>
  <div
    className="content"
  >
    这是cartoon的
    文字
  </div>
  <div />
  <p
    style={
      Object {
        "fontFamily": "keAi1",
        "fontSize": "24px",
      }
    }
  >
    猫和老鼠
  </p>
  <p
    style={
      Object {
        "fontFamily": "keAi2",
        "fontSize": "36px",
      }
    }
  >
     —— 汤姆和杰瑞
  </p>
  <p
    style={
      Object {
        "fontFamily": "keAi3",
        "fontSize": "48px",
      }
    }
  >
    可可爱爱的朋友
  </p>
  <div
    className="flex-row mt-50"
  >
    <img
      className="image"
      src="test-file-stub"
    />
    <img
      className="image"
      src="test-file-stub"
    />
    <img
      className="image"
      src="test-file-stub"
    />
  </div>
  <div
    className="flex-row mt-50"
  >
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
    <img
      className="icon"
      src="test-file-stub"
    />
  </div>
</section>
`;
